packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.test.tsx

```diff
@@ -279,31 +280,57 @@ describe('routes/Product/SubscriptionCreate', () => {
     };
   }
 
+  enum CustomerType {
+    NoCustomer,
+    WithCustomer,
+    WithCustomerCard,
+    IapCustomerOnly,
+  }
```

Comment:
Do we still need all this?

Reply:
Yup we do. Each of the test cases that call `commonPaymentSubmissionTest` need a different Customer object.